GetOwnershipClaimStatus/handler.ts

```diff
@@ -16,27 +18,116 @@ import {
 } from "@pagopa/ts-commons/lib/strings";
 import { ContextMiddleware } from "@pagopa/io-functions-commons/dist/src/utils/middlewares/context_middleware";
 import { RequiredParamMiddleware } from "@pagopa/io-functions-commons/dist/src/utils/middlewares/required_param";
+import * as t from "io-ts";
+import { NumberFromString } from "@pagopa/ts-commons/lib/numbers";
+import * as TE from "fp-ts/TaskEither";
+import { flow, pipe } from "fp-ts/lib/function";
+import { DatabaseError, Pool } from "pg";
 import { ClaimProcedureStatus } from "../generated/definitions/ClaimProcedureStatus";
+import { IConfig, IDecodableConfigPostgreSQL } from "../utils/config";
+import { queryDataTable } from "../utils/db";
+import {
+  IDbError,
+  toPostgreSQLError,
+  toPostgreSQLErrorMessage
+} from "../models/DomainErrors";
+
+export const ResultRow = t.interface({
+  count: NumberFromString,
+  status: t.string
+});
+export type ResultRow = t.TypeOf<typeof ResultRow>;
+
+export const ResultSet = t.interface({
+  rowCount: t.number,
+  rows: t.readonlyArray(ResultRow)
+});
+export type ResultSet = t.TypeOf<typeof ResultSet>;
+
+export const createSql = (dbConfig: IDecodableConfigPostgreSQL) => (
+  organizationFiscalCode: OrganizationFiscalCode,
+  sourceId: NonEmptyString
+): NonEmptyString =>
+  `SELECT status, count(status)
+	FROM "${dbConfig.DB_SCHEMA}"."${dbConfig.DB_TABLE}"
+	WHERE "organizationFiscalCode" = '${organizationFiscalCode}' and "sourceId" = '${sourceId}'
+	GROUP BY status` as NonEmptyString;
```

Comment:
tryo to use `knex`, so that we get special character escaping (other that sql injection prevention)

Reply:
Thanks, I did a first try with `knex` with 73db150